mod-email/src/main/resources/domain/mod-email/features/delayedTasks.feature

```diff
@@ -5,14 +5,22 @@ Feature: Delayed tasks
     * callonce login testUser
     * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
 
-  @Undefined
   Scenario: Delete should return 204 on success
-    * print 'undefined'
+    Given path 'delayedTask/expiredMessages'
+    When method DELETE
+    Then status 204
 
-  @Undefined
   Scenario: Delete should return 400 if bad request
```

Comment:
if invalid token?

Reply:
I didn't find another way to get 400

Reply:
then I would suggest renaming the scenario

Reply:
In which name do you suggest rename this test?